src/assets/viselements.json

```diff
@@ -904,6 +904,16 @@
             "type": "str",
             "doc": "Allows the styling of table cells.\nSee <a href=\"#styling\">below</a> for details."
           },
+          {
+            "name": "tooltip",
+            "type": "str",
+            "doc": "The name of the function that must return a tooltip text for a cell.\nSee <a href=\"#cell%20tooltip\">below</a> for details."
```

Comment:
How does that reflect in Studio?

Reply:
It is reflected as string/markdown string --> will only rendered as a normal string